src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub use mount::{Mount, Mountable};
 pub use favicon_handler::FaviconHandler;
 pub use default_error_handler::DefaultErrorHandler;
 pub use json_body_parser::JsonBody;
-pub use query_string::{QueryString, Query};
```

Comment:
It's true that both of these are handling urlencoded things but I think it's slightly confusing to have to think about that when navigating the code. I'd prefer to capture the similarities within the urlencoded module but still leave `Query` and `FormBody` in their own modules.